src/workflow/workflow.py

```diff
@@ -1151,27 +1372,28 @@ def version(self):
 
             version = None
             # environment variable has priority
-            if self.alfred_env.get('workflow_version'):
-                version = self.alfred_env['workflow_version']
+            if self.alfred_env.get("workflow_version"):
+                version = self.alfred_env["workflow_version"]
 
             # Try `update_settings`
             elif self._update_settings:
-                version = self._update_settings.get('version')
+                version = self._update_settings.get("version")
 
             # `version` file
             if not version:
-                filepath = self.workflowfile('version')
+                filepath = self.workflowfile("version")
 
                 if os.path.exists(filepath):
-                    with open(filepath, 'rb') as fileobj:
+                    with open(filepath, "rb") as fileobj:
```

Comment:
Regex `match_version` in `update.py` will fail when given a binary version string like `"b'v8.0'"`. Reading as a text file resolves this.

```suggestion
                    with open(filepath, "r") as fileobj:
```